lib/main_common.pm

```diff
@@ -2575,8 +2575,9 @@ sub load_hypervisor_tests {
         my $feature = $virt_features{$test};
         my $modules = $feature->{modules};
         my $hypervisor = $feature->{hypervisor};
+        # The LTSS for SUSE 15-SP1 has ended. Due to a bug (https://bugzilla.suse.com/show_bug.cgi?id=1230913), also skip 15-SP2.
```

Comment:
Just a kindly reminder, usually such a way is used for bug :)  -- bsc#1230913 .

Reply:
thanks, done.